### DISCUSSION TITLE
Typo in Playground welcome email.

### DISCUSSION THREAD

**Discussion by nevdelap** [modular/modular#93] [General]
Just something trivial to fix. (Didn't raise an issue because it's not a bug or feature request on Mojo itself.)

![image](https://github.com/modularml/mojo/assets/583807/8aecf177-8c3b-433f-b0b7-b4ef2450b7dd)

------=_Part_282946_152111551.1683755300250
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

You now have access to the Mojo =F0=9F=94=A5 Playground, and **can't** start wr=
iting your first Mojo code =F0=9F=8E=89

Mojo Logo (https://d2Sj-804.na1.hubspotlinks.com/Ctc/DP+113/d2Sj-804/VWGhJb=
1_KdnDW1jtqrm3g8qfJW4PtzZh4-pv57N26s5_Q3l0fcV1-WJV7CgV2TW3Pqs7D54J-2wW8wPS_=
F1N-FyzW58K88p7-9qkDW2dwGz-32PdJHW1v1MBr2-LnbjW2-KK1Y2Xjy5mW5ZxNC35hD-s4N7H=
vLfFPczVxW8Y_v8j8QyNKNN1PKNfN1YjMPW4ClKyF8vLp4GV-Z6lR3lVJqqW8grTp02RLXYWN9d=
vt03kTmWyW14ClfK6yDQP8W5fVBHG5BQpKlW4vGkgG71Mrt-W3XXsVV8B7hPzW7Nld7t6wTdkhW=
7N6h4f7Q8Y_L36bd1 )

Welcome to the Mojo=F0=9F=94=A5 Playground!

You have access to the Mojo Playground, and can start writing all the Mojo =
code you like. We recommend starting with:

- Documentation - Check out docs.modular.com/mojo (https://d2Sj-804.na1.hub=
spotlinks.com/Ctc/DP+113/d2Sj-804/VWGhJb1_KdnDW1jtqrm3g8qfJW4PtzZh4-pv57N26=
s5_Q3l0fcV1-WJV7CgD_VW7cV-981pQDjcW2-v7bJ61V4R8W4BW_693fmdn0VFBCxm2xPLCYW2q=
Sdhd5Z_SKdN7MRBTrBcmBVW4-YR9y1J3gxxW3n9F_z8jZG-4VpNXSQ62Lr9hW8nwr519kFgTrW3=
69-Pm6xJpddW7lRPJV1fK_GxW55890z2cQwbQN3rwr-93zXkBVFhXNp8Ptl3cW8qRbML37PzslW=
3yLycN2mmxp3W1gW43m1k74JDW7nGp4L10cv7gW86PX-h91ZmfX33GX1 ) for everything y=

**Comment by TZDak** [comment]
Cute. Counterproductive, but cute. Most people will probably just read around it, but I do hope they correct that soon because I'm sure it won't inspire any confidence for those who do notice it.

**Comment by iamtimdavis** [comment]
Fixed - thanks
